code/style.css

```diff
@@ -1,5 +1,83 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@300;400&display=swap');
 
 body {
-  background-color: lightblue;
+    background-color: rosybrown;
+    font-family: 'Raleway', sans-serif;
+    color: whitesmoke;
+    margin: 0 60px;
 }
+
+header {
+    background-image: url("./img/background-peony.jpg");
+    background-size: cover;
+    height: 300px;
+    margin: 20px 0;
+    border: 2px solid whitesmoke;
+    padding: 15px;
+}
+
+menu{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+menu a {
+    display: inline-block;
+    padding: 10px 15px;
+    background: white;
+    color: lightcoral;
+    font-weight: bold;
+    text-decoration: none;
+    border-radius: 3px;
+    margin: 0 5px;
+}
+
+menu a:hover{
+    background: lightcoral;
+    color: white;
+}
+
+h1 {
+    text-align: left;
+    font-size: 50px;
+    margin: 20px;
+    font-family: 'Raleway', sans-serif;
+    text-shadow: 1px 1px gray;
+}
+
+h2 {
+    font-family:'Raleway', sans-serif;
+}
+
+.main-block {
+    border: 2px dotted lightcoral;
+    padding: 10px;
+    margin-bottom: 20px;
+}
+
+.highlighter-block {
+    background-color: #c8a3a3;
+    display: inline-block;
+    padding: 10px;
+    border-radius: 20px;
+    margin: 5px;
+    box-shadow: 0px 0px 10px 1px whitesmoke;
+}
+
+.note {
+    font-style: italic;
+}
+
+a{
```

Comment:
Really small thing, but remember to have space between element and curly bracket: `a {`